pom.xml

```diff
@@ -1,40 +1,47 @@
-<?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
     <parent>
         <groupId>org.springframework.boot</groupId>
         <artifactId>spring-boot-starter-parent</artifactId>
-        <version>2.7.4</version>
+        <version>2.7.2</version>
         <relativePath/> <!-- lookup parent from repository -->
     </parent>
     <groupId>com.starbucks</groupId>
-    <artifactId>starbucks</artifactId>
+    <artifactId>com.starbucks</artifactId>
     <version>0.0.1-SNAPSHOT</version>
-    <name>starbucks</name>
-    <description>starbucks</description>
+    <name>Controller</name>
+    <description>Demo project for Spring Boot</description>
     <properties>
-        <java.version>17</java.version>
+        <java.version>11</java.version>
     </properties>
     <dependencies>
         <dependency>
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-web</artifactId>
+            <version>2.7.4</version>
```

Comment:
You can skip version if you have a version of the spring-boot-starter-parent and it's 2.7.2 and here you're using 2.7.4 they are working well with one version.

Reply:
It's a great opportunity.